common/setups/rasr/hybrid_system.py

```diff
@@ -482,11 +487,25 @@ def nn_recognition(
                     native_lstm_job.out_op,
                     forward_output_layer,
                 )
+                tf_flow.config.tf_fwd.loader.saved_model_file = checkpoints[epoch]
+                self.tf_flows[f"{name}-{epoch}"] = tf_flow
+
                 flow = self.add_tf_flow_to_base_flow(feature_flow, tf_flow)
                 flow.config.tf_fwd.loader.saved_model_file = checkpoints[epoch]
 
+                self.feature_scorers[recognition_corpus_key][
+                    f"pre-nn-{name}-{prior:02.2f}"
+                ] = scorer
+                self.feature_flows[recognition_corpus_key][
+                    f"{feature_flow_key}-tf-{epoch:03d}"
+                ] = flow
```

Comment:
@christophmluscher is this used lateron?

Note that `self.feature_flows[recognition_corpus_key]` can also be a `rasr.FlowNetwork` already and not a dict, see a couple of lines above. I get an error due to this
```
TypeError: 'FlowNetwork' object does not support item assignment
```